python/composio/tools/toolset.py

```diff
@@ -10,12 +10,14 @@
 import os
 import time
 import typing as t
+import uuid
 import warnings
 from datetime import datetime
 from functools import wraps
 from importlib.util import find_spec
 from pathlib import Path
 
+from flask import session
```

Comment:
The `flask.session` import appears to be unused in this file. Consider removing it if it's not needed elsewhere in the codebase.